src/apis/issue.ts

```diff
@@ -129,6 +131,10 @@ export class DefaultIssueAPI implements IssueAPI {
         return pagedIterator<IssueRequest>(this.api.query.issue.issueRequests, perPage);
     }
 
+    async getIssuePeriod(): Promise<BlockNumber> {
+        return (await this.api.query.issue.getIssuePeriod() as BlockNumber);
```

Comment:
Shouldn't it be `return await this.api.query.issue.issuePeriod();` Best would be to add an integration test to verify that the call is correct here: https://github.com/interlay/polkabtc-js/blob/dev/test/integration/apis/issue.test.ts